fec.config.js

```diff
@@ -1,27 +1,31 @@
-const { resolve } = require('path');
+const path = require("path");
+const { dependencies, insights } = require("./package.json");
+
+const moduleName = insights.appname.replace(/-(\w)/g, (_, match) =>
+    match.toUpperCase()
+);
+
+const srcDir = path.resolve(__dirname, "./src");
 
 module.exports = {
-    appUrl: '/insights/patch',
-    debug: true,
-    useProxy: process.env.PROXY === 'true',
-    proxyVerbose: true,
-    plugins: [],
-    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
-    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    appName: moduleName,
+    appUrl: "/insights/patch",
+    useProxy: process.env.PROXY === "true",
     moduleFederation: {
+        moduleName,
```

Comment:
I don't see this eg in compliance, is this needed?

Reply:
This is how the moduleName was listed in my app, and the template I was given to implement. 
Please site if there is better documentation somewhere.

Reply:
which is now resolved correct with the last commit :) resolving the thread